client/src/App.tsx

```diff
@@ -1,45 +1,16 @@
 import React from 'react';
-import { Redirect, Route } from 'react-router-dom';
-import { IonApp, IonRouterOutlet } from '@ionic/react';
-import { IonReactRouter } from '@ionic/react-router';
-import Task from './pages/Tasks';
-import AddTask from './pages/AddTask';
-import UpdateTask from './pages/UpdateTask';
-import OfflineQueue from './pages/OfflineQueue';
+import { Router } from './components';
+import { keycloakHelpers } from './helpers';
 
-/* Core CSS required for Ionic components to work properly */
-import '@ionic/react/css/core.css';
+// load all styles
+import './theme';
 
-/* Basic CSS for apps built with Ionic */
-import '@ionic/react/css/normalize.css';
-import '@ionic/react/css/structure.css';
-import '@ionic/react/css/typography.css';
+export const App: React.FC = () => {
 
-/* Optional CSS utils that can be commented out */
-import '@ionic/react/css/padding.css';
-import '@ionic/react/css/float-elements.css';
-import '@ionic/react/css/text-alignment.css';
-import '@ionic/react/css/text-transformation.css';
-import '@ionic/react/css/flex-utils.css';
-import '@ionic/react/css/display.css';
+  // flush undefined tokens from local storage
+  keycloakHelpers.flushInvalidTokens();
```

Comment:
What I like in this approach is that we have separate file to manage all things. 
Really good job on this instead of polluting codebase with some keycloak!